modules/getting-started-openshift-gitops.adoc

```diff
@@ -0,0 +1,128 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="getting-started-with-red-hat-openshift-gitops_{context}"]
+= Getting Started with Red Hat OpenShift GitOps
+
+[id="installing-red-hat-openshift-gitops_{context}"]
+== Installing Red Hat OpenShift GitOps
+
+.Prerequisite
+
+The Red Hat product version of Argo CD is the Openshift GitOps Operator. When you install the Red Hat OpenShift GitOps operator, you get an instance of Argo CD in the 'openshift-gitops' namespace.
+
+To ensure you can work with the Red Hat OpenShift GitOps operator, go to xref:../operators/admin/olm-adding-operators-to-cluster.adoc#adding-operators-to-a-cluster[installing an operator from OperatorHub].
+
+After you install an Operator, you have a Red Hat OpenShift Container cluster. Continue with the default operator installation because this operator installs to all namespaces on the cluster. Now you have installed a Red Hat OpenShift GitOps operator.
+
+[id="logging-into-argocd_{context}"]
+== Logging into ArgoCD
+
+The Red Hat OpenShift GitOps operator automatically sets up a ArgoCD cluster configuration. ArgoCD manages specific cluster-scoped resources, including: platform operators, optional Operator Lifecycle Manager (OLM) operators, and user management. Because ArgoCD is not a cluster-admin, you must launch into the ArgoCD instance from the Console Application Launcher.
+
+To find the password for the ArgoCD:
```

Comment:
Add this as the second step: Enter the password to log into the ArgoCD UI.
And then say - To find the password for the ArgoCD:
Steps on line 26 to 31 should all be nested steps under the above step.